src/util/load.js

```diff
@@ -1,9 +1,14 @@
 import { readdirSync, readJSONSync } from './modules/fs'
 import minimatch from 'minimatch'
+import { aliases } from './aws/api-gateway'
 
-export function envs () {
-  return readdirSync('environments')
-  .map((file) => file.split('.').shift())
+export async function envs () {
+  const api = this.api()
+  if (api && api.id) {
+    return aliases(api.id)
+  } else {
+    return []
```

Comment:
list your own stage or it won't do anything right now. you can deploy to an arbitrary stage. just realized i have to update the inquirer logic everywhere we use `load.envs` in the yargs handlers